.github/workflows/dotnet-core.yml

```diff
@@ -35,13 +36,49 @@ jobs:
 
       - name: Determine Version and Update csproj Files
         uses: gittools/actions/gitversion/execute@v0.9.9
+        id: gitversion
         with:
           useConfigFile: true
           additionalArguments: '/updateprojectfiles'
     
       - name: Display updated csproj File
         run: cat Freshli/Freshli.csproj
 
+      # Required by GitReleaseManager.  Hopefully not for long.
```

Comment:
I wonder if you could put all this into a separate Github Action job (like eclint, etc.) that would only run on `main` or on a release branch so we're not building the `dotnet-core.yml` file to be huge and can run this in parallel.

Reply:
Haha.  I was just updating my initial PR comment with the same thought.  The only issue is I would need to duplicate the GitVersion steps, which is not horrible would be nice to avoid.  Do you know of a why to reuse steps in different workflow files?